src/components/CartModal/CartModal.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import CartIcon from '../../assets/images/shopingCart.jpg';
+import CartList from './CartList';
+
+function MyVerticallyCenteredModal(props) {
+    let totalPrice = 0;
+    let regEx = /\D/g;
+
+    props.props.cartItems.forEach((item, i) => {
+        let price = item.price.replace(regEx, '');
+        totalPrice += +price * +props.props.cartItems[i].count;
```

Comment:
```suggestion
        totalPrice += +price * +props.props.cartItems[i].count;
```
Array count je uvek number ne treba ti konverzija `+props...`